.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "common-dev-asset"]
+	path = common-dev-asset
+	url = git@github.com:terraform-ibm-modules/common-dev-assets.git
```

Comment:
I think we should use https because schematics does not support ssh yet. That is what we did internally anyway

Reply:
fixed